### PR TITLE
Add trace exporter example and fix product name from Stackdriver to Cloud Trace

### LINKED ISSUE #10: Add trace exporter example
Add trace exporter from opentelemetry-go

### PR DESCRIPTION
**Pull request by ymotongpoo**
Fixes #10. (Merge #8 before this PR)

### CODE REVIEW

File: example/http/client/client.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	"io/ioutil"
+	"log"
+	"os"
+
+	"net/http"
+	"time"
+
+	"google.golang.org/grpc/codes"
+
+	cloudtrace "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
```
Comment (rghetia) [inline example/http/client/client.go]: nit: s/cloudtrace/texpoter/

File: example/http/go.mod [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module github.com/GoogleCloudPlatform/opentelemetry-operations-go/
```
Comment (rghetia) [inline example/http/go.mod]:
```suggestion
module github.com/GoogleCloudPlatform/opentelemetry-operations-go/example/http
```

File: example/http/server/server.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+package main
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	cloudtrace "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
```
Comment (rghetia) [inline example/http/server/server.go]: nit: s/cloudtrace/texporter/
or exportert

File: exporter/trace/cloudtrace_test.go [hunk 4]
```diff
@@ -24,14 +24,14 @@ import (
 	"testing"
 	"time"
 
+	cloudtrace "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
```
Comment (rghetia) [inline exporter/trace/cloudtrace_test.go]: s/cloudtrace/texporter/

**Comment by rghetia** [comment]
@ymotongpoo please rebase as there are conflicts with #8.

**Comment by ymotongpoo** [comment]
@rghetia I think the conflicts are now solved.

**Pull request merged**